code/game/objects/structures/barsigns/bay.dm

```diff
@@ -112,3 +112,33 @@
 /datum/barsign/bay/bark
 	name = "The Bark"
 	icon = "bark"
+
+/datum/barsign/bay/topmen
+	name = "Top Men"
+	icon = "topmen"
+	pixel_x = -16
+	pixel_y = -31
```

Comment:
Offsetting by negative pixels causes the icon to be resized by those dimensions. So for example, you can make a 48x48 icon occupy 32x32 space in-game by offsetting its `pixel_x` and `pixel_y` by -16.